tasks/object-detection-yolov8/main.py

```diff
@@ -138,9 +138,20 @@ def getPatience(taskRun: TaskRun) -> int:
     return 2 ** 64
 
 
+def verifySplitValid(validationSplit: float, datasetSize: int) -> None:
```

Comment:
This function should return a bool, not raise an exception in case something went wrong